src/ethereum/forks/gas_repricing/utils/constant.py

```diff
@@ -0,0 +1,145 @@
+"""
+Gas Cost Constants for Gas Repricing Fork.
+
+This module provides a unified dataclass for accessing gas costs
+loaded from the gas_cost.yaml configuration file.
+
+Reference: https://github.com/ethereum/execution-specs/issues/1599
```

Comment:
The specs themselves are always _the reference_. Any relevant information should be moved into this file.